etna/core/templates/wagtailadmin/home.html

```diff
@@ -0,0 +1,4 @@
+{% extends "wagtailadmin/home.html" %}
+
+
+{% block branding_welcome %}Welcome to the etna {% if FEATURE_PLATFORM_ENVIRONMENT_TYPE == "staging" or FEATURE_PLATFORM_ENVIRONMENT_TYPE == "development" %}DEVELOP {% endif %}Wagtail CMS{% endblock %}
```

Comment:
Suggestion: Instead of checking that it is either `staging` or `development`. Why not just check it's not `production`?  
```suggestion
{% block branding_welcome %}Welcome to the etna {% if FEATURE_PLATFORM_ENVIRONMENT_TYPE != "production" %}DEVELOP {% endif %}Wagtail CMS{% endblock %}
```

Note: I'm not 100% sure of the prod' naming convention so just double check.

Reply:
Hi Will, cheers for the suggestions - I agree that this is a better way to do it syntactically, although, it may cause some issues. Where I set up the .env variable in `base.py`, I made the default be `""`. I did this because I didn't want to manually assign it to a "default" environment.
This means that, if the `os.getenv` fails, it will be given `""` which will then default to the standard Wagtail styling. If I made the suggestions here, I would then have to make it default to `"production"`, which isn't a bad thing.

What are your thoughts on changing the default value to `"production"` instead of `""`?

Reply:
Personally, the default should be a production environment, a developer should have to configure via env' vars for others. How that's done doesn't particularly matter so your approach does work above. However, you can alter that conditional statement to allow for the behaviour just checking if there's a value, otherwise show the production version. 

I'll leave that choice to you. Feel free to merge.